### PR TITLE
Feature/25

### PR DESCRIPTION
**Pull request by alelucena**
Resolved errors (changed typedefs) and improved user validation. For ease, merge feature 25 as it should contain both features 24 and 25.

### CODE REVIEW

File: server/graphql/resolvers/user.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+const { AuthenticationError } = require('apollo-server');
+const User = require('../../models/User');
+const bcrypt = require('bcrypt');
+//const { MONGODB } = require('../../../config');
+//var MongoClient = require("mongodb").MongoClient;
+//var url = MONGODB
```
Comment (DanielCufino) [inline server/graphql/resolvers/user.js:6]: remove unnecessary commented lines

File: server/graphql/resolvers/user.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+Mutation: {
+    createUser: async (_, { netID }, context) => {
+        
+        const newUser = new User ({
+            netID, first_name, middle_initital, last_name,
+             password, email, payment, college,
+        });
+
+        newUser.email = email
```
Comment (DanielCufino) [inline server/graphql/resolvers/user.js:18]: I'd recommend adding some form of user email validation. 

Trying looking into this resourcce: https://www.abstractapi.com/tools/email-regex-guide

Also I'd also recommend validating by checking the email ends in `@rice.edu`

**Review by DanielCufino — CHANGES_REQUESTED** [review]
Looking good just a couple small things

**Comment by alelucena** [comment]
Please review branch again.

**Review by DanielCufino — APPROVED** [review]
LGTM!

**Comment by DanielCufino** [comment]
Please merge changes

**Pull request merged**